src/sub-blocks/BasicCard/BasicCard.tsx

```diff
@@ -37,7 +33,7 @@ const BasicCard = (props: BasicCardProps) => {
             className={b()}
             contentClassName={b('content')}
             {...cardParams}
-            extraProps={{'aria-describedby': descriptionId, 'aria-labelledby': titleId}}
+            extraProps={{role: 'group'}}
```

Comment:
Why the card (especially [card with url](https://preview.gravity-ui.com/page-constructor/980/?path=/story/components-cards-basiccard--with-url)) should have `group` role? If one needs the `group` role specified, then it should be possible to specify it via props, e.g. via `extraProps`.